.azure_pipelines/scripts/ClientPatch.py

```diff
@@ -0,0 +1,107 @@
+"""Patching ADO Client to retrieve continuation token.
+
+Related to question in following issue:
+https://github.com/microsoft/azure-devops-python-api/issues/461
+
+"""
+
+import logging
+from typing import Optional, cast
+
+from azure.devops import _models
```

Comment:
maybe you can add a requirements.txt file for the dependencies to run these scripts